### DISCUSSION TITLE
在SwiftBackup使用Webdav上次同名文件会报错

### DISCUSSION THREAD

**Discussion by uvwt** [AlistGo/alist#6854] [Q&A]
在手机上使用webdav时，上传同名文件会不成功。例如在Swift Backup备份应用数据时，应用数据发生变化时，再次上次应用数据会上传不成功，因为原文件已存在，删除云盘webdav里面的文件
就可以上传成功了，怎么解决？
![Screenshot_2024-07-23-00-22-08-770_org swiftapps swiftbackup](https://github.com/user-attachments/assets/470c9baa-09cc-4961-b669-6b74c60585a1)

**Comment by pongfcnkl** [comment]
写个脚本，在上传前定时删掉老文件

**Reply by uvwt** [reply]
不能直接覆盖掉吗😭，写脚本太复杂了

**Reply by pongfcnkl** [reply]
> 不能直接覆盖掉吗😭，写脚本太复杂了

不对，你这个不是覆盖的问题，是webdav客户端超时了（或者其他问题），经过网页上复制，是可以覆盖的。
建议写脚本备份，反正你也root了，先备份到本地，然后在用rclone同webdav名称加timeout参数，这样上传就稳定了
